### ISSUE TITLE
[Docs Needed] Explain what is bluetooth absolute mouse

### ISSUE THREAD

**Issue by landall** [sobrinho/ESP32-BLE-Abs-Mouse#7]
I think it is necessary to explain what is a bluetooth absolute mouse.
This can help the coders not to use a wrong library.

**Comment by vanyasem** [comment]
This thread seems helpful: https://forum.arduino.cc/t/ble-hid-mouse-with-cursor-absolute-positioning/1216238

**Comment by vanyasem** [comment]
**TLDR:** "Absolute mouse" refers to absolute positioning of the cursor on the screen (similar to a light gun), contrary to the usual relative mouse positioning

Unless you need to point the mouse at the exact absolute coordinates on the screen, you need to use https://github.com/T-vK/ESP32-BLE-Mouse instead of this library

As the support for platforms is rather low in this library (only Android is supported), you might want to look into USB Mouse emulation, as it has official Absolute Mouse positionong in ESP-IDF: https://github.com/espressif/arduino-esp32/commit/bd39fcfac610ef74313f0cb5bed9523ae69d2262

**Comment by vanyasem** [comment]
If you're interested in Absolute Mouse positioning over bluetooth, take a look at https://github.com/asterics/esp32_mouse_keyboard/issues/30
There is an experimental implementation in this branch: https://github.com/asterics/esp32_mouse_keyboard/tree/absoluteMouse
